src/products/components/ProductVariantCreatorPage/ProductVariantCreatorPage.tsx

```diff
@@ -69,61 +64,54 @@ function canHitNext(
       }
 
       return true;
-    case "summary":
+    case ProductVariantCreatorStep.summary:
       return data.variants.every(variant => variant.sku !== "");
 
     default:
       return false;
   }
 }
 
-export interface ProductVariantCreateDialogProps
+export interface ProductVariantCreatePageProps
   extends Omit<
-    ProductVariantCreateContentProps,
+    ProductVariantCreatorContentProps,
     "data" | "dispatchFormDataAction" | "step" | "onStepClick"
   > {
   defaultPrice: string;
-  open: boolean;
-  onClose: () => void;
   onSubmit: (data: ProductVariantBulkCreateInput[]) => void;
 }
 
-const ProductVariantCreateDialog: React.FC<
-  ProductVariantCreateDialogProps
-> = props => {
-  const {
-    attributes,
-    defaultPrice,
-    errors: apiErrors,
-    open,
-    onClose,
-    onSubmit,
-    ...contentProps
-  } = props;
-  const classes = useStyles(props);
-  const [step, setStep] = React.useState<ProductVariantCreateStep>("values");
-
-  function handleNextStep() {
-    switch (step) {
-      case "values":
-        setStep("prices");
-        break;
-      case "prices":
-        setStep("summary");
-        break;
-    }
+function getTitle(step: ProductVariantCreatorStep, intl: IntlShape): string {
+  switch (step) {
+    case ProductVariantCreatorStep.values:
+      return intl.formatMessage({
+        defaultMessage: "Choose Values",
+        description: "product attribute values, page title"
+      });
+    case ProductVariantCreatorStep.prices:
+      return intl.formatMessage({
+        defaultMessage: "Price and SKUs",
+        description: "page title"
+      });
+    case ProductVariantCreatorStep.summary:
+      return intl.formatMessage({
+        defaultMessage: "Summary",
+        description: "page title"
+      });
   }
+}
 
-  function handlePrevStep() {
-    switch (step) {
-      case "prices":
-        setStep("values");
-        break;
-      case "summary":
-        setStep("prices");
-        break;
-    }
-  }
+const ProductVariantCreatePage: React.FC<ProductVariantCreatePageProps> = props => {
+  const { attributes, defaultPrice, errors, onSubmit, ...contentProps } = props;
+  const classes = useStyles(props);
+  const intl = useIntl();
+  const [step, { next, prev, set: setStep }] = useWizard<
```

Comment:
Could we use
nextStep
prevStep
setStep
as standard? next/prev/set is hard to track and could be anything (for example next item in the loop)

Reply:
Or if we dont pay per character then maybe `navigateNextStep` would be really clear.